src/main.js

```diff
@@ -1,10 +1,10 @@
 // ==UserScript==
 // @name        BCA GPA
 // @description GPA Calculator for BCA
-// @match       https://ps001.bergen.org/*
+// @match       https://bcts.powerschool.com/*
 // ==/UserScript==
 
-if (document.URL === "https://ps001.bergen.org/guardian/home.html") {
+if (document.URL === "https://bcts.powerschool.com/guardian/home.html") {
```

Comment:
Doesn't work if user signs in through the Student Sign In page. This adds `?_userTypeHint=student` to the end of the URL, so could you update this to account for any URL parameters?

Reply:
updated and tested [with edge chromium]